src/interfaces/index.ts

```diff
@@ -10,11 +8,6 @@ export * from './store-interfaces'
 export * from './utils-interfaces'
 export * from './state-interface'
 
-// this export is needed for some of the shared components
-export interface ActionableItemIF {
-  action: ActionTypes
-}
```

Comment:
The shared component still needs this, but has it available in the shared component library itself.